src/main/java/com/spartronics4915/frc2020/RobotContainer.java

```diff
@@ -170,10 +170,11 @@ private void configureJoystickBindings()
         new JoystickButton(mJoystick, 1).toggleWhenPressed(mLauncherCommands.new ShootBallTest(mLauncher));
         new JoystickButton(mJoystick, 2).toggleWhenPressed(mLauncherCommands.new TurretTest(mLauncher));
         new JoystickButton(mJoystick, 3).toggleWhenPressed(mLauncherCommands.new HoodTest(mLauncher));
-        new JoystickButton(mJoystick, 7).whileHeld(new TrajectoryTrackerCommand(mDrive, mDrive,
+         new JoystickButton(mJoystick, 7).whileHeld(new TrajectoryTrackerCommand(mDrive, mDrive,
             this::throughTrench, mRamseteController, mStateEstimator.getEncoderRobotStateMap()));
         new JoystickButton(mJoystick, 7).whileHeld(new TrajectoryTrackerCommand(mDrive, mDrive,
             this::toControlPanel, mRamseteController, mStateEstimator.getEncoderRobotStateMap()));
+        new JoystickButton(mJoystick, 3).toggleWhenPressed(mLauncherCommands.new AutoAimTurret(mLauncher,Constants.Launcher.goalLocation,mStateEstimator.getEncoderRobotStateMap()));
```

Comment:
@binnur All of the commands in `configureJoystickBindings()` were for testing and should now be commented out in `master`. 

The dashboard right now (for testing) is set up so that Ethan can live adjust the RPS, angle, and rotation of the launcher without having to redeploy code. Those methods toggle whether any of the devices should be running based on those values.

Reply:
Although, we do use `toggleWhenPressed` for the intake's Harvest and Eject functions. I can ask by Cruz whether he'd prefer a separate start / stop intaking buttons but we're running somewhat low.